README.md

```diff
@@ -1,63 +1,64 @@
 
-The nature of the project one is working on really defines which version control system is best for that specific project's needs. However, in this article, we will be focusing on the GitHub environment.
+The nature of the project one is working on really defines which version control system is best for that specific project's needs ([ref?]()). However, in this article, we will be focusing on the GitHub environment.
 
 ## Advantages of using a Version Control System 
 
-Using a source code version control system is essential in modern-day software development practices. These tools offer powerful features and advantages to teams working on a code project.
+~~Using a source code version control system is essential in modern-day software development practices. These tools offer powerful features and advantages to teams working on a code project.~~
 
-Here's a list of a few features that a version control system can offer for your project:
-- It provides the ability to supervise and protect the codebase (__Source:__ [Simplilearn](https://www.simplilearn.com/tutorials/devops-tutorial/version-control)).
-- As already mentioned, it gives developers the ability to record all changes made to the code and save the code in those particular states (__Source:__ [Simplilearn](https://www.simplilearn.com/tutorials/devops-tutorial/version-control)).
-- By using a version control tool, a developer can analyze version history, giving them the power to revisit a code in any of the previous saved states (__Source:__ [Simplilearn](https://www.simplilearn.com/tutorials/devops-tutorial/version-control))
-- Not only does it provide a feature to revisit code, but it also allows developers to roll back the code to a previous state.
+~~Here's a list of a few features that a version control system can offer for your project:~~
+- ~~It provides the ability to supervise and protect the codebase (__Source:__ [Simplilearn](https://www.simplilearn.com/tutorials/devops-tutorial/version-control)).~~
+- ~~As already mentioned, it gives developers the ability to record all changes made to the code and save the code in those particular states (__Source:__ [Simplilearn](https://www.simplilearn.com/tutorials/devops-tutorial/version-control)).~~
+- ~~By using a version control tool, a developer can analyze version history, giving them the power to revisit a code in any of the previous saved states (__Source:__ [Simplilearn](https://www.simplilearn.com/tutorials/devops-tutorial/version-control))~~
+- ~~Not only does it provide a feature to revisit code, but it also allows developers to roll back the code to a previous state.~~
 - All these features give the developers an environment to experiment and make errors without affecting the final working code.
 - All these updates and experiments can easily be reviewed when using a version control tool.
```

Comment:
These last two statements are new. I would suggest changing this section to focus on these two points, because they are both valuable. If you wrote a paragraph for each then that would certainly constitute a clear set of advantages to the use of VCS.